ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -4,31 +4,45 @@
     step: pre_deploy
   ansible.builtin.import_playbook: ./hooks.yml
 
+- name: Perform Podified and EDPM deployment on compute nodes provisioned with bmaas
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Call  edpm_deploy_baremetal role
+      when: edpm_compute_baremetal | default('false') | bool
+      ansible.builtin.include_role:
+        name: edpm_deploy_baremetal
+
 - hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
     - name: Call predeploy
+      when: not edpm_compute_baremetal | default('false') | bool
       ansible.builtin.include_role:
         name: edpm_prepare
```

Comment:
Can we merge these two plays like this (since both are going to run on the same host)?
```
- hosts: "{{ cifmw_target_host | default('localhost') }}"
  gather_facts: false
  tasks:
    - name: name: Perform Podified and EDPM deployment on compute nodes provisioned with bmaas
      when: edpm_compute_baremetal | default('false') | bool
      ansible.builtin.include_role:
        name: edpm_deploy_baremetal

    - name: Deploy podified control plane
      when: not edpm_compute_baremetal | default('false') | bool
      ansible.builtin.include_role:
        name: edpm_prepare
```

Reply:
thanks for suggestion, done